packages/experimental/Button/src/ButtonColorTokens.ts

```diff
@@ -23,7 +23,7 @@ export const defaultButtonColorTokens: TokenSettings<ButtonTokens, Theme> = (t:
     pressed: {
       backgroundColor: t.colors.defaultPressedBackground,
       color: t.colors.defaultPressedContent,
-      borderColor: t.colors.defaultPressedBorder,
+      borderColor: t.colors.defaultHoveredBorder,
```

Comment:
Is this intended to change to Hovered for the `pressed` state?

Reply:
Oh no, thanks for catching :)